server/sockets/list.js

```diff
@@ -107,11 +109,25 @@ const updateItemState = (socket, itemClientLocks) => {
 
 const updateItem = socket => {
   socket.on(ItemActionTypes.UPDATE_SUCCESS, data => {
-    const { listId } = data;
+    const { listId, userId } = data;
+    const sanitizedUserId = sanitize(userId);
+    let editedBy;
+
+    User.findById(sanitizedUserId)
+      .then(user => {
+        if (user) {
+          const { displayName } = user;
+          editedBy = displayName;
+        }
+      })
+      .finally(() => {
```

Comment:
Please use `then` - `finally` should be used as a cleanup for error handling. In case of missing user - I would throw an error as this is not an expected situation.